alpine: Add support for armv7 Alpine Linux

nghiant2710 opened the pull request:
binary built from Debian Jessie will be compatible with OpenSSl1.0.x and
from Debian Stretch will be compatible with OpenSSL1.1.x

Signed-off-by: Trong Nghia Nguyen <nghiant2710@gmail.com>

nghiant2710 committed:
build: Set different file name for binary built from Debian Jessie and Stretch

binary built from Debian Jessie will be compatible with OpenSSl1.0.x and
from Debian Stretch will be compatible with OpenSSL1.1.x

Signed-off-by: Trong Nghia Nguyen <nghiant2710@gmail.com>

petrosagg commented on build.sh:
```diff
@@ -10,8 +8,23 @@ gpg --keyserver ha.pool.sks-keyservers.net --recv-keys 0D96DF4D4110E5C43FBFB17F2
+
 
 mkdir -p /usr/src/python
 curl -SL "https://www.python.org/ftp/python/${PYTHON_VERSION%%[a-z]*}/Python-$PYTHON_VERSION.tar.xz" -o python.tar.xz
```
You should download the key using its keyid from a key server here

nghiant2710 replied:
@petrosagg all the keys used to verify are downloaded before https://github.com/balena-io/python-build/blob/5128a9d14375a2a4101facada36b4545d0a55898/build.sh#L3-L6

nghiant2710 committed:
alpine: Add support for armv7 Alpine Linux

Since armv7 Alpine Linux is released, the build script is updated to support this architecture

Change-type: patch
Signed-off-by: Trong Nghia Nguyen <nghiant2710@gmail.com>

nghiant2710 committed:
Merge branch 'master' into master
